pkg/kv/transport_race.go

```diff
@@ -57,7 +57,9 @@ func GRPCTransportFactory(
 	opts SendOptions, rpcContext *rpc.Context, replicas ReplicaSlice, args roachpb.BatchRequest,
 ) (Transport, error) {
 	if atomic.AddInt32(&running, 1) <= 1 {
-		rpcContext.Stopper.RunWorker(context.TODO(), func(ctx context.Context) {
+		// NB: We can't use Stopper.RunWorker because doing so would race with
+		// calling Stopper.Stop.
+		rpcContext.Stopper.RunAsyncTask(context.TODO(), "transport racer", func(ctx context.Context) {
```

Comment:
Did you mean to ignore the error return from `RunAsyncTask`?

Reply:
Not intentionally. I wonder why lint or errcheck didn't warn me about that.